### ISSUE TITLE
Detect if hidden or visible

### ISSUE THREAD

**Issue by Norman191** [ravindu1024/android-keyboardlistener#5]
I Followed your Answer from **_stackoverflow_**.
 i have a question :

=> while using this 

    KeyboardUtils.addKeyboardToggleListener(spalsh.this, new 
    KeyboardUtils.SoftKeyboardToggleListener() {
                @Override
                public void onToggleSoftKeyboard(boolean isVisible) {
    Toast.makeText(this, "keyboard visible = " + isVisible, Toast.LENGTH_SHORT).show();

            }

        });

it show true and false directly.
How can i control what to do when Keyboard is visible ?
Using if and else .

something like : 

                 if (KEYBOARD is visible){
                  //// visible do something

                  } else { 
                /// hidden do something

                }

**Comment by ravindu1024** [comment]
Yes, correct. I didn't want to add 2 separate functions just to differentiate between true and false so both states are returned in the same call. Just do an if check. 

Btw, in your code, is `spalsh` the name of your activity or is that a reference to it?

**Comment by Norman191** [comment]
Thanks i fixed it.

How Can i add the checker in the **_BroadcastReceiver_** to keep checking in bg if other apps opened the keyboard.

    KeyboardUtils.addKeyboardToggleListener(THIS, new KeyboardUtils.SoftKeyboardToggleListener() {
            @Override
            public void onToggleSoftKeyboard(boolean isVisible) {
                Toast.makeText(context, "keyboard visible = " + isVisible, Toast.LENGTH_SHORT).show();
            }
        });

i got the context error, i tried to add a Context, but not working
